src/gbafix.c

```diff
@@ -141,7 +141,7 @@ int main(int argc, char *argv[])
 	if (argc <= 1)
 	{
 		printf("GBA ROM fixer v"VER" by Dark Fader / BlackThunder / WinterMute \n");
-		printf("Syntax: gbafix <rom.gba> [-p] [-t[title]] [-g<game_code>] [-m<maker_code>] [-v<version>]\n");
+		printf("Syntax: gbafix <rom.gba> [-p] [-t[title]] [-c<game_code>] [-m<maker_code>] [-r<version>]\n");
```

Comment:
This should be a separate PR, it's not relevant to this change.

Reply:
Sorry, I noticed that the usage screen was wrong in testing my change, so I figured it would be okay to include it here as it's such a small change. I'll split it into its own.